src/components/tick-slider.jsx

```diff
@@ -0,0 +1,65 @@
+import Styles from "../styles/slider.module.scss";
+import { createEffect, createSignal } from "solid-js";
+
+const Slider = (props) => {
+  const [day_mid, setDayMid] = createSignal("");
+  const [rotate, setRotate] = createSignal("rotate(180deg)");
+  const [data, setData] = createSignal({ left: 0, right: 0 });
+
+  const getData = () => {
+    const value = props.ticks;
+    if (value < day_mid()) {
+      setData({ left: parseFloat(value), right: 0 });
+      setRotate("rotate(0deg)");
+    } else if (value > day_mid()) {
+      setData({ left: 0, right: parseFloat(value) });
+      setRotate("rotate(180deg)");
+    }
+  };
+
+  createEffect(() => {
+    const mid_value = (props.day_low + props.day_high) / 2;
+    setDayMid(mid_value.toFixed(props.step_value));
+    getData();
+  });
+
+  return (
+    <div>
+      <span class={Styles.span}>Daily range</span>
+      <div>
+        <input
+          type="range"
+          id="input-left"
+          class={Styles.slider1}
+          min={props.day_low}
+          max={day_mid()}
+          value={data().left}
+          step={1 / Math.pow(10, props.step_value)}
+          style={{ transform: rotate() }}
+        />
+        <input
+          type="range"
+          id={Styles.range}
+          min={day_mid()}
+          max={props.day_high}
+          class={Styles.slider2}
+          value={data().right}
+          step={1 / Math.pow(10, props.step_value)}
+        />
+        <div class={Styles.indicator}>
+          <div>
+            <div class={Styles.arrow_down} />
```

Comment:
We have stylings for tick arrow in watchlist.module.scss, Can u see if u can reuse that?

Reply:
we can check this only once the slider component is used inside any pages